apis/projectcontour/v1/detailedconditions.go

```diff
@@ -113,4 +113,20 @@ type DetailedCondition struct {
 	Warnings []SubCondition `json:"warnings,omitempty"`
 }
 
-const ValidConditionType string = "Valid"
+const (
+	ValidConditionType string = "Valid"
```

Comment:
may as well make this declaration consistent with the other ones, i.e. omit `string`

Reply:
removed